docs/rules/no-test-only.md

```diff
@@ -0,0 +1,37 @@
+# uvu/no-test-only
+
+Warn about forgotten `test.only()`.
+
+## Rationale
+
+It is popular mistake to forget `test.only()` from debug session.
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```javascript
+test.only("sum", () => {
+  tassert.type(math.sum, "function");
```

Comment:
Let’s remove 15-18 and 26-29 lines to focus on important.

Use `() => {}` like in other rules.